### ISSUE TITLE
Ephemeral nodes not being deleted

### ISSUE THREAD

**Issue by DentonGentry** [tailscale/tailscale#3779]
### What is the issue?

There have been several reports of ephemeral nodes not being deleted in a timely manner. I created an ephemeral node which hasn't been deleted after 24 hours of inactivity: 100.102.53.65.

The unit test for ephemeral nodes is still passing, so something else at a higher layer is apparently going wrong.

### OS version

N/A

### Tailscale version

N/A

**Comment by praneetloke** [comment]
I thought I had this problem when an ephemeral VM in my network wouldn't disappear from the Tailscale app on my Android device. Then I realized I wasn't connected to my Tailnet. Just thought I'd mention that because I haven't run into an issue with ephemeral nodes being removed the list since realizing that. It definitely wasn't 24 hours for me. I am on the latest version 1.20.1(?) for Linux. On Android I joined the beta program so it might be on a newer version. I haven't launched an ephemeral VM in about a week now so maybe this is something new with the code behind the Tailnet?
